libbindgen/src/ir/context.rs

```diff
@@ -545,7 +545,7 @@ impl<'ctx> BindgenContext<'ctx> {
     }
 
     fn build_root_module(id: ItemId) -> Item {
-        let module = Module::new(Some("root".into()));
+        let module = Module::new(Some("root".into()), /* inline = */ false);
```

Comment:
Nitpick: the need for a comment in `/* inline = */ false)` is a code smell, pointing us towards a two-member enum:

```rust
enum ModuleKind {
    Normal, // OutOfLine? Maybe something based off of wording from the standard?
    Inline,
}
```